loom/engine/bindings/loom/lmGameController.cpp

```diff
@@ -0,0 +1,124 @@
+#include "loom/engine/bindings/loom/lmGameController.h"
+#include "loom/common/core/log.h"
+
+GameController GameController::controllers[MAX_CONTROLLERS];
+
+lmDefineLogGroup(controllerLogGroup, "loom.controller", 1, LoomLogInfo);
+
+GameController::GameController() : is_connected(false), gamepad(0), instance_id(-1), haptic(0) {}
+
+void GameController::open(int device)
+{
+    lmLogInfo(controllerLogGroup, "Opening device [%d]", device);
+
+    // OUTPUT GUID OF DEVICE FOR TESTING //
```

Comment:
Debug probably, but I will most likely remove it now, or at least comment it out.